.github/workflows/CI.yml

```
@@ -18,7 +18,17 @@ permissions:
       - "**/*.gitignore"
       - .editorconfig
       - docs/**
-  pull_request: null
+  pull_request:
+    branches:
+      - main
+    tags-ignore:
```

actionlint (1.7.10)

"tags-ignore" filter is not available for pull_request event. it is only for push event

(events)